lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -19,10 +20,11 @@ def initialize(serializer, options = {})
 
         def serializable_hash(options = nil)
           options ||= {}
+
           if serializer.respond_to?(:each)
-            serializable_hash_for_collection(serializer, options)
+            serializable_hash_for_collection(options)
           else
-            serializable_hash_for_single_resource(serializer, options)
```

Comment:
Not required for this PR, but passing the serializer reference along is useless here as it is available via an `attr_reader`.